site/package.json

```diff
@@ -17,7 +17,7 @@
     "@types/mark.js": "^8.11.12",
     "autoprefixer": "^10.4.16",
     "postcss": "^8.4.32",
-    "tailwindcss": "^3.4.0",
+    "tailwindcss": "^3.3.3",
```

Comment:
You're using minor release flag `^3.3.3` which results in installing `3.4.0`. 
`pnpm.lock.yaml`
<img width="351" alt="Screenshot 2024-03-22 at 12 49 34 PM" src="https://github.com/solidjs-community/solid-primitives/assets/29286430/a0c15ffb-df40-4eb1-957e-794a4bf6bd8d">
You need to install this specific version, without any flags `3.3.3`.
`site/package.json`
```json
"tailwindcss": "3.3.3",
```

Commands to fix, remove tailwindcss depency, then install it with specific version of `3.3.3`, make sure to run inside `site` directory.
```bash
pnpm remove tailwindcss && pnpm add tailwindcss@3.3.3 -D
```

Reply:
Thanks for your help, the command worked. Sorry that it took so long, I have been working on another pr in the meanwhile. The branch should be now ready to merge